sycl/unittests/buffer/MemChannel.cpp

```diff
@@ -97,6 +97,12 @@ TEST_F(BufferMemChannelTest, MemChannelProp) {
 
   sycl::queue Q{Plt.get_devices()[0]};
   sycl::buffer<int, 1> Buf(3, sycl::property::buffer::mem_channel{42});
+
+  EXPECT_TRUE(Buf.has_property<sycl::property::buffer::mem_channel>());
```

Comment:
could the ASSERT_TRUE be better here? does the rest of test have any meaning if property is absent?

Reply:
Originally I used that, but I figured since it worked before without these queries working it would be fine to with `EXPECT`. On the other hand, the queries failed to compile before, so maybe you're right that `ASSERT` is better. I'll change it.

Reply:
Done in 36c77db80b5021813bcf86fdd2d82fe8ff1a09f8.